Client/package.json

```diff
@@ -20,6 +20,7 @@
     "@mui/x-data-grid": "7.3.2",
     "@mui/x-date-pickers": "7.3.2",
     "@reduxjs/toolkit": "2.2.5",
+    "autosuggest-highlight": "^3.3.4",
```

Comment:
What is this package?  We shouldn't need to add any libraries this component, MUI should have everything we need.

Introducing more packages makes the app larger and more vulnerable so we should do so only if we absolutely need to

Reply:
This package what in the doc itself! without this, the dropdown on the autocomplete does not work!

Reply:
The `Autocomplete` Component is a wrapper itself. What lets you type and to get your options is simply a Text Field that is passed to the `renderInput`. The doc used the MUI's `TextField` for in its example which made it unlikely to change in its styles, neither our Field component could be passed to it! So, I passed an `input` element to it to make it as light and editable as possible.

I had to wrap it inside Stacks to add validations `errors`, `helperText`, and `title`, in case we want to have such things added in the future.

Reply:
> This package what in the doc itself! without this, the dropdown on the autocomplete does not work!

The example in the docs for a controlled component doesn't seem to require that?

```
import * as React from 'react';
import TextField from '@mui/material/TextField';
import Autocomplete from '@mui/material/Autocomplete';

const options = ['Option 1', 'Option 2'];

export default function ControllableStates() {
  const [value, setValue] = React.useState(options[0]);
  const [inputValue, setInputValue] = React.useState('');

  return (
    <div>
      <div>{`value: ${value !== null ? `'${value}'` : 'null'}`}</div>
      <div>{`inputValue: '${inputValue}'`}</div>
      <br />
      <Autocomplete
        value={value}
        onChange={(event, newValue) => {
          setValue(newValue);
        }}
        inputValue={inputValue}
        onInputChange={(event, newInputValue) => {
          setInputValue(newInputValue);
        }}
        id="controllable-states-demo"
        options={options}
        sx={{ width: 300 }}
        renderInput={(params) => <TextField {...params} label="Controllable" />}
      />
    </div>
  );
}
```

Reply:
> I had to wrap it inside Stacks to add validations `errors`, `helperText`, and `title`, in case we want to have such things added in the future.

I don't think we need any of those things here :thinking:   I think we can safely remove them and simplify here